src/image.ml

```diff
@@ -106,6 +115,7 @@ value print_source_image conf f =
   let fname =
     if f.[0] = '/' then String.sub f 1 (String.length f - 1) else f
   in
+  (* TODO remove Filename.basename to handle sub_dirs *)
```

Comment:
OK, but it is better to create an issue instead of putting some TODO in the code.